app/src/generated/graphql.schema.json

```diff
@@ -1603,6 +1603,16 @@
         "description": "",
         "fields": null,
         "inputFields": [
+          {
```

Comment:
It might be nice to exclude these from version control and generate them on build for clarity. Down to chat about wiring that up at some point if you're open to it?

Reply:
Yea, let's chat about it (definitely possible, and favorable) - added story here https://trello.com/c/eQI8i9qK/91-stop-checking-in-generated-files